### DISCUSSION TITLE
Recognize now

### DISCUSSION THREAD

**Discussion by sotoniak** [blakeblackshear/frigate#5438] [Ask A Question]
Hi, is there any way to trigger Frigate from Home Assistant to ask if a car is now in the zone, for example?

The aim is to find out if there is a car in the parking lot at any time.

**Comment by blakeblackshear** [comment]
There isn't a way to do this, but why wouldn't frigate already be tracking those cars? It scans the entire frame on startup and keeps trying to track parked cars unless you have configured it to stop tracking.
